From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: way-zer <himc.wicp@gmail.com>
Date: Fri, 19 Apr 2024 12:27:50 +0800
Subject: [PATCH] OC: no MapPreviewLoader.setupLoaders

---
 core/src/mindustry/ClientLauncher.java | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/core/src/mindustry/ClientLauncher.java b/core/src/mindustry/ClientLauncher.java
index f19fa20cc0d6910a0fa3658092849e9e673f8659..6b69c2d175dd47163842f15d1e8148751d4f78e5 100644
--- a/core/src/mindustry/ClientLauncher.java
+++ b/core/src/mindustry/ClientLauncher.java
@@ -147,7 +147,8 @@ public abstract class ClientLauncher extends ApplicationCore implements Platform
         assets.load("sprites/error.png", Texture.class);
         atlas = TextureAtlas.blankAtlas();
         Vars.net = new Net(platform.getNet());
-        MapPreviewLoader.setupLoaders();
+        //MDTX: optimize performance (inspired by Foo)
+//        MapPreviewLoader.setupLoaders();
         mods = new Mods();
         schematics = new Schematics();
 
