From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: way-zer <himc.wicp@gmail.com>
Date: Thu, 3 Jul 2025 22:45:51 +0800
Subject: [PATCH] H MetricCollector.handleException

---
 core/src/mindustry/net/CrashHandler.java | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/core/src/mindustry/net/CrashHandler.java b/core/src/mindustry/net/CrashHandler.java
index 1b52810d28edf893950f9609952e0f9a704e44da..f1c7c258da1d13921bfbd2c8415269c27c9bc59d 100644
--- a/core/src/mindustry/net/CrashHandler.java
+++ b/core/src/mindustry/net/CrashHandler.java
@@ -9,6 +9,7 @@ import arc.util.io.*;
 import mindustry.*;
 import mindustry.core.*;
 import mindustry.mod.Mods.*;
+import mindustryX.features.*;
 
 import java.io.*;
 import java.text.*;
@@ -20,6 +21,8 @@ import static mindustry.Vars.*;
 public class CrashHandler{
 
     public static String createReport(Throwable exception){
+        MetricCollector.INSTANCE.handleException(exception);
+
         String error = writeException(exception);
         LoadedMod cause = getModCause(exception);
 
@@ -47,6 +50,7 @@ public class CrashHandler{
         try{
             Core.settings.getDataDirectory().child("crashes").child("crash_" + System.currentTimeMillis() + ".txt")
             .writeString(createReport(exception));
+            MetricCollector.INSTANCE.waitPost();//MDTX:Wait before exiting.
         }catch(Throwable ignored){
         }
     }
@@ -111,6 +115,7 @@ public class CrashHandler{
             death.printStackTrace();
         }
 
+        MetricCollector.INSTANCE.waitPost();//MDTX:Wait before exiting.
         System.exit(1);
     }
 
